interfaces/policy/helpers.go

```diff
@@ -26,16 +26,33 @@ import (
 	"github.com/snapcore/snapd/asserts"
 	"github.com/snapcore/snapd/release"
 	"github.com/snapcore/snapd/snap"
+	"github.com/snapcore/snapd/strutil"
 )
 
 // check helpers
 
-func checkSnapType(snapType snap.Type, types []string) error {
+func snapIDSnapd(snapID string) bool {
+	var snapIDsSnapd = []string{
+		// production
+		"PMrrV4ml8uWuEUDBT8dSGnKUYbevVhc4",
+		// TODO: when snapd snap is uploaded to staging, replace this with
+		// the real snap-id.
+		"todo-staging-snapd-id",
+		// tests
+		"snapd-snap-id",
```

Comment:
do we need this one here?  can't we use one of the other in tests?

Reply:
We could use one of the other ones.

Reply:
Done now.